impress-org/Google-Places-Reviews
# New: Google Places Reviews Gutenberg block

## Description
This PR implements a new Gutenberg block for displaying Google Places Reviews. The code is updated as well and some minor bugs are resolved. Also, Laravel Mix is now used for compiling assets, and all dependencies are updated.


## Visuals

![Screen Capture_select-area_20211223180657](https://user-images.githubusercontent.com/4222590/147273325-1b5366de-08bd-408e-905c-32ea5aff88a2.gif)

## Testing Instructions

Add a new Google Places Review block and play with options. Also, test everything on the frontend to make sure that everything is working correctly. 

## Pre-review Checklist
